test/test_utils/imageutils.py

```diff
@@ -0,0 +1,100 @@
+import re
+import boto3
+import json
+
+def get_image_account_id(image_uri):
```

Comment:
We already have an equivalent function in test_utils/__init__.py.

https://github.com/aws/deep-learning-containers/blob/5f02453540ccf56862355b6864a746a2765fd4ed/test/test_utils/__init__.py#L1091-L1098

Is this meant to replace that function? Should it the other function be removed and replaced everywhere else?

Reply:
It is meant to factor out all helper functions used for image. In the future, we can refactor the code to use imageutil module and remove helper functions that appear in other places.